pkg/libxcomposite

```diff
@@ -5,18 +5,24 @@ libx11
 libxfixes
 
 [vars]
-filesize=283746
-sha512=b15f8fdbe4cb7488bdad161d73ef8e59ef603c5af92d7b573110ba136d7d4ce3749aac45972bce87662b0310b67e831cffad72c45c15bfff988b5b555d4c884b
+filesize=12469
+sha512=f88547243ca387d5093e61299c398d119c031470e9290d3af3393cf3237b2d5d8a2657565474c5ef9265c5b45a2d5bf0428f630d4a2d4bc1c1174b0d5f6a7c42
+pkgver=2
 
 [mirrors]
-http://xorg.freedesktop.org/releases/individual/lib/libXcomposite-0.4.4.tar.bz2
+https://gitlab.freedesktop.org/xorg/lib/libxcomposite/-/archive/libXcomposite-0.4.6/libxcomposite-libXcomposite-0.4.6.tar.bz2
```

Comment:
please use `https://www.x.org/releases/individual/lib/libXcomposite-0.4.6.tar.xz` which comes with configure script, so autoconf doesnt have to be added. while at it, can you move mirrors section to top ?